ckanext/ontario_theme/templates/internal/snippets/facet_arrangement.html

```diff
@@ -3,11 +3,12 @@
 {% elif h.lang() == 'fr' %}
   {% set keywords = "keywords_en" %}
 {% endif %}
-{% set excluded_facets = [keywords] %}
-
+{% set excluded_facets = [keywords, "access_level"] %}
+{% set fields = h.scheming_get_dataset_schema('dataset')['dataset_fields'] %}
+ 
 {# Flag facets to be ignored from standard loop that calls facets. #}
 {% set single_option_facets = ["organization_jurisdiction", "organization_category"] %}
-
+{% do excluded_facets.append(single_option_facet) if single_option_facet else none %}
 {% if single_option_facets %}
```

Comment:
No need to check `if single_option_facets` since they are hard-coded above in line 10